### ISSUE TITLE
FEATURE: Can see list of drivers with cell phone numbers

### ISSUE THREAD

**Issue by pconrad** [ucsb-cs156/proj-gauchoride#45]
# Depends on

At least the first two stories from Epic #62 (any user can add their cell phone) must be completed first; i.e the ones that add cell phone to the entity, and add a backend route for updating the cell phone.

# User Story

* As a rider, admin, or driver
* I can see a list of all drivers, along with their cell phone numbers
* So that I can contact drivers as needed

# Discussion

The spreadsheet from Mike Fogelsanger from 05/17/23 showed tables of both riders and drivers with cell phone numbers.

It isn't yet clear who should have access to the cell phone numbers, but for now, let's assume that everyone that has been designated as a driver, rider, or admin can see all of the entries in the table.

# Acceptance Criteria

- [x] There is a `/drivers/list` page that shows a list of all drivers with their names, emails and  where available, their cell phone numbers.

# Implementation Todos

Each of these could be a separate issue/PR, or they could be combined.  But keep in mind that separate smaller PRs usually accumulate more points are get reviewed and merged faster.

- [x] Add backend DriversController with a route `/api/drivers/all` that gets a list of all users that are drivers.
- [x] Add a navbar element for Driver List as a top level button (for now) that routes to a page `/drivers`, which could be a placeholder at first (i.e. a DriverListPage component.)
- [x] Create a DriverTable component (and storybook entry) to list drivers with name, email, cell phone
- [x] Add the DriverTable to the DriverListPage.

**Comment by Mohamed-Elfouly** [comment]
Was partially implemented from m23-9am-2
